### PR TITLE
Feature/add ue integration docs

### PR DESCRIPTION
**Pull request by rarcif**

### CODE REVIEW

File: docs/play/getting-started_cpp.md [hunk 1]
```diff
@@ -23,10 +23,10 @@ Please download the archive file based on your OS in the [release page](https://
 
 #### Windows
 
-1. Clone the current repository
-   ```sh
-   git clone https://github.com/cronos-labs/play-cpp-sdk.git
-   ```
```
Comment (damoncro) [inline docs/play/getting-started_cpp.md:29]: May I know why remove the indent?

Original:
image
Now:

Comment (damoncro) [inline docs/play/getting-started_cpp.md:29]: May I know why remove the indent?

Original:
image
Now:
image

Comment (rarcif) [reply docs/play/getting-started_cpp.md:29]: We can put the indent back as before, but then I have to change it on the other pages as well to make it consistence. What do you think?

Comment (rarcif) [reply docs/play/getting-started_cpp.md:29]: @damoncro fixed

**Review by damoncro — APPROVED** [review]

**Review by lezzokafka — APPROVED** [review]

**Pull request merged**